CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased]
 
+### Added
```

Comment:
Please add an entry mentioning that the digital data can now be accessed publicly